scripts/generate_sitemap/index.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'path';
+import { concat } from 'lodash';
+import { simpleSitemapAndIndex } from 'sitemap';
+import {
+  getLearnPageItems,
+  getTopLevelPageItems,
+  getLocationPageItems,
+  getHomeImageItems,
+} from './utils';
+
+const destinationDir = path.join(__dirname, '../../public');
```

Comment:
Are you intending that we check this file in?  Or do we generate it as part of our deploy process?  If the latter, I think it should probably go in `../../build` instead?

Reply:
I think we need to generate it as part of the deploy process - it needs to be available under `https://covidactnow.org/` for it to work, so I think it needs to go in `../../build`